javascript/javascript1/week2/Js-Week2/main.js

```diff
@@ -0,0 +1,100 @@
+// const balance =13300;
+// if (balance <= 0) {
+//   console.log("Please deposit some money");
+// } else if (balance > 0 && balance <= 1000) {
+//   console.log("Your balance is looking okay");
+// } else if (balance > 1000 && balance <= 3000) {
+//   console.log("Your balance is looking good");
+// } else if (balance > 3000 && balance <= 10000) {
+//   console.log('Your balance is fantastic');
+// }else if (balance > 10000) {
+//   console.log('Your balance is AMAZING!');
+// }
+
+
+//*Event application*
+
+const weekday = [
+  "Sunday",
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+];
+function getEventWeekday(date) {
+  const eventDay = (new Date().getDay() + date) % 7;
+  const eventWeekday = weekday[eventDay];
+  return ` ${date}  ${eventWeekday}.`;
+}
+console.log(getEventWeekday(9));
+
+// *Flight booking fullname function*
+
+function GetFullname(firstName, surName, useFormalName) {
+  this.firstName = firstName;
+  this.surName = surName;
+  this.FullName = () => `${this.firstName} ${this.surName}`;
+  this.useFormalName = () =>
+    useFormalName === true
+      ? `Lord ${this.firstName} ${this.surName}`
+      : `${this.firstName} ${this.surName}`;
+}
+const getFullname = new GetFullname("Jack", "jensen", true);
+console.log(getFullname.FullName());
+console.log(getFullname.useFormalName());
+
+// *Weather wear*
+
+function clothesToWear(temp) {
+  if (temp <= 0) {
+    return "Weather is Cold: Wear A winter Jacket.";
+  } else if (temp > 0 && temp <= 10) {
+    return "weather is Not too Cold: Wear sweatshirt.";
+  } else if (temp > 10 && temp <= 20) {
+    return "Weather is warm today: Wear Jeans.";
+  } else {
+    return "Weather is hot today: Wear T-shirt.";
+  }
+}
+console.log(clothesToWear(12));
+
+// *Student manager*
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (class07Students.length < 6 || studentName === "") {
+    if (class07Students.includes(studentName)) {
+      console.log(true);
+    } else if (studentName === "") {
```

Comment:
You are checking `studentName === "" `twice. 

Consider restructuring the function:

If the name string is empty, log a meaningful warning and return undefined;

If the Student array is already 6 students, add if it is the Queen, otherwise, log a meaningful warning and return undefined;

If the name is provided, check if it is already in the class, if yes, log a meaningful warning and return the original array, if no, add the student.